### PR TITLE
Fixed spelling and adjusted white spaces

### PR DESCRIPTION
**Pull request by pawan**
### Description (*)

   Corrected spelling and adjust white space to increase readability   

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lbajsarowicz — APPROVED** [review]
Please fix Static tests, then PR will be automatically approved.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6474](https://jira.corp.magento.com/browse/ENGCOM-6474) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@pawankparmar thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by pawan** [comment]
> Please fix Static tests, then PR will be automatically approved.

Hello @lbajsarowicz , do I need to **phpcs** OR **phpcbf** file ?

**Comment by lbajsarowicz** [comment]
Fix your code to pass static tests

**Comment by lbajsarowicz** [comment]
Maybe I was unclear about the expectations.
What you actually have to do is fix these:
image

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 1]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
+     * @param  \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
      * @return $this
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function setExtensionAttributes(
         \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
     );
-}
+}
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There has to be an empty line at the end of file.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 2]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
      * Set an extension attributes object.
      *
-     * @param \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
+     * @param  \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
      * @return $this
-     * @since 101.1.0
+     * @since  101.1.0
      */
     public function setExtensionAttributes(
         \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Import the class, make it single line.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 3]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
     public function getExtensionAttributes();
 
     /**
      * Set an extension attributes object.
      *
-     * @param \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
+     * @param  \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
      * @return $this
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 4]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
      */
     public function getExtensionAttributes();
 
     /**
      * Set an extension attributes object.
      *
-     * @param \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
+     * @param  \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface $extensionAttributes
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 5]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
+     * @since  101.1.0
      */
     public function setRegularPrice($regularPrice);
 
     /**
      * Retrieve existing extension attributes object or create a new one.
      *
      * @return \Magento\Catalog\Api\Data\ProductRender\FormattedPriceInfoExtensionInterface|null
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 6]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
     public function getRegularPrice();
 
     /**
      * Set regular price
      *
-     * @param string $regularPrice
+     * @param  string $regularPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 7]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
...
      */
     public function getRegularPrice();
 
     /**
      * Set regular price
      *
-     * @param string $regularPrice
+     * @param  string $regularPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 8]
```diff
@@ -133,35 +133,35 @@ public function setMinimalPrice($minimalPrice);
      * Usually this price is corresponding to price in admin panel of product
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 9]
```diff
@@ -105,26 +105,26 @@ public function getMinimalRegularPrice();
...
     public function getSpecialPrice();
 
     /**
      * Set minimal price
      *
-     * @param string $minimalPrice
+     * @param  string $minimalPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 10]
```diff
@@ -105,26 +105,26 @@ public function getMinimalRegularPrice();
...
      */
     public function getSpecialPrice();
 
     /**
      * Set minimal price
      *
-     * @param string $minimalPrice
+     * @param  string $minimalPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 11]
```diff
@@ -105,26 +105,26 @@ public function getMinimalRegularPrice();
...
+     * @since  101.1.0
      */
     public function setSpecialPrice($specialPrice);
 
     /**
      * Retrieve special price
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 12]
```diff
@@ -105,26 +105,26 @@ public function getMinimalRegularPrice();
      *
      * Special price - is temporary price, that can be set to specific product
      *
-     * @param string $specialPrice
+     * @param  string $specialPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 13]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
      */
     public function setMinimalRegularPrice($minRegularPrice);
 
     /**
      * Retrieve minimal regular price
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 14]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
     public function getMaxRegularPrice();
 
     /**
      * The minimal regular price has the same behavior of calculation as max regular price, but is opposite price
      *
-     * @param string $minRegularPrice
+     * @param  string $minRegularPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 15]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
      */
     public function getMaxRegularPrice();
 
     /**
      * The minimal regular price has the same behavior of calculation as max regular price, but is opposite price
      *
-     * @param string $minRegularPrice
+     * @param  string $minRegularPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 16]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
+     * @since  101.1.0
      */
     public function setMaxRegularPrice($maxRegularPrice);
 
     /**
      * Retrieve max regular price
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 17]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
      * Set max regular price
-     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogules
+     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalog rules
      * in it
      *
-     * @param string $maxRegularPrice
+     * @param  string $maxRegularPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 18]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
...
     public function getMinimalPrice();
 
     /**
      * Set max regular price
-     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogules
+     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalog rules
      * in it
      *
-     * @param string $maxRegularPrice
+     * @param  string $maxRegularPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 19]
```diff
@@ -60,43 +60,43 @@ public function setMaxPrice($maxPrice);
      * The minimal price is for example, the lowest price of all variations for complex product
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 20]
```diff
@@ -42,16 +42,16 @@ public function setFinalPrice($finalPrice);
...
     public function getMaxPrice();
 
     /**
      * Set the max price of the product
      *
-     * @param string $maxPrice
+     * @param  string $maxPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 21]
```diff
@@ -42,16 +42,16 @@ public function setFinalPrice($finalPrice);
...
      */
     public function getMaxPrice();
 
     /**
      * Set the max price of the product
      *
-     * @param string $maxPrice
+     * @param  string $maxPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 22]
```diff
@@ -42,16 +42,16 @@ public function setFinalPrice($finalPrice);
      * E.g. for product with custom options is price with the most expensive custom option
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 23]
```diff
@@ -23,17 +23,17 @@ interface FormattedPriceInfoInterface extends \Magento\Framework\Api\ExtensibleD
...
 
     /**
      * Set the final price: usually it calculated as minimal price of the product
      * Can be different depends on type of product
      *
-     * @param string $finalPrice
+     * @param  string $finalPrice
      * @return void
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 24]
```diff
@@ -23,17 +23,17 @@ interface FormattedPriceInfoInterface extends \Magento\Framework\Api\ExtensibleD
...
      */
     public function getFinalPrice();
 
     /**
      * Set the final price: usually it calculated as minimal price of the product
      * Can be different depends on type of product
      *
-     * @param string $finalPrice
+     * @param  string $finalPrice
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php [hunk 25]
```diff
@@ -23,17 +23,17 @@ interface FormattedPriceInfoInterface extends \Magento\Framework\Api\ExtensibleD
      * Retrieve html with final price
      *
      * @return string
-     * @since 101.1.0
+     * @since  101.1.0
```
Comment (lbajsarowicz) [inline app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: There should be only one space.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 1 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 2 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 3 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 4 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 5 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 6 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 7 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 9 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 10 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 11 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 12 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 13 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 14 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 15 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 16 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 17 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 18 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 19 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 20 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 21 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 22 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 23 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 24 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

File: app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php (hunk 25 above)
Comment (pawan) [reply app/code/Magento/Catalog/Api/Data/ProductRender/FormattedPriceInfoInterface.php]: Fixed.

**Comment by pawan** [comment]
> Maybe I was unclear about the expectations.
> What you actually have to do is fix these:
> image

Hello @lbajsarowicz , thank you pointing this.

I have fixed other space issue and new line at the end of file.
Still I couldn't fix "**There must be exactly one blank line between lines**" issue.

In this pull, I just correct spelling and put space between two words!
I am not getting why static test is failing

I have tried following test for my file:
`phpcs --standard=PHPCS /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=Squiz /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=PEAR /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=Zend /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=MySource /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=PSR2 /home/comp1/Desktop/FormattedPriceInfoInterface.php`
`phpcs --standard=PSR1 /home/comp1/Desktop/FormattedPriceInfoInterface.php`

   None of the above reported "**There must be exactly one blank line between lines**" issue

Thank you.

**Comment by pawan** [comment]
@lbajsarowicz 

> Please fix Static tests, then PR will be automatically approved.

I have fixed Static tests...

**Comment by pawan** [comment]
@akaplya @lbajsarowicz any updates on PR ? do i need to anything else ?

**Comment by pawan** [comment]
@magento-engcom-team, any updates on PR ? anything remaining from my side ?

**Review by lbajsarowicz — APPROVED** [review]
OK

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6474](https://jira.corp.magento.com/browse/ENGCOM-6474) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**